src/EditProvider.ts

```diff
@@ -99,16 +100,84 @@ export class EditProvider
   }
 
   public static beautifyOptions(
-    filePath: string = vscode.workspace.rootPath
+    path: string = vscode.workspace.rootPath
```

Comment:
Is this a file path, directory path, or both?

Reply:
It is both, depending on what a call is passing to the function ()